purchase_requisition_from_rfq/models/purchase_requisition.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2019 Ecosoft Co., Ltd (http://ecosoft.co.th/)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+from odoo import _, fields, models
+from odoo.exceptions import ValidationError
+
+
+class PurchaseRequisition(models.Model):
+    _inherit = "purchase.requisition"
+
+    purchase_ids = fields.One2many(
+        default=lambda self: self._context.get("active_ids", []),
+    )
+    line_ids = fields.One2many(
+        default=lambda self: self._get_line(),
+    )
+    from_rfq = fields.Boolean(
+        default=False,
+        readonly=True,
+        help="True, if this purchase agreement is created from RFQs",
+    )
+
+    def _get_line(self):
```

Comment:
`active_ids` was used in 2 positions, I think should declare variable `active_ids`

active_ids = self._context.get("active_ids", [])

Reply:
Fixed.